kiwipy/rmq/threadcomms.py

```diff
@@ -117,6 +119,18 @@ def __init__(self,
                 decoder=decoder,
                 testing_mode=testing_mode))
 
+    def start(self):  # pylint: disable=no-self-use
+        raise DeprecationWarning("This method was deprecated in v0.6.0 and will be removed in 0.7.0.  There is no "
+                                 "replacement and the class should be reinstantiated if it's been closed.")
+
+    @deprecation.deprecated(deprecated_in="0.6.0",
+                            removed_in="0.7.0",
+                            current_version=__version__,
+                            details="Use close() instead.  Unlike stop(), close() is permanent and "
+                            "the class should not be used again after this.")
```

Comment:
Why use this decorator for `stop` and normal raise for `start`?

Reply:
Well there's nothing sane I can do in `start()` other than raise so if the code reaches that point it can only crash.  What I did forget is that static analysis tools could still forewarn someone...I'll add in the decorator as well.

Reply:
Technically, that is not "deprecating" though, that is just removing it. Any code that was calling it explicitly, will now simply break. But okay, if there is no alternative, then we just leave it like this